src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,27 @@
-export const TodoInfo = () => {};
+import { Todo } from "../../types/Todo";
+import { UserInfo } from "../UserInfo";
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
```

Comment:
Directly importing `usersFromServer` here violates the requirement to prepare data in the `App` component and pass it down. Consider refactoring to receive the necessary user data as props from a parent component, ensuring data is managed in a single place.